internal/cocoa/notIOS.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2022 The Ebitengine Authors
```

Comment:
ditto: please use small cases. `notios.go` is fine. 

Unfortunately it might be hard to change only cases of a file name. You might have to remove this file once, commit the change, then add the file again.